post/2017/11-12-speed-release.md

```diff
@@ -0,0 +1,62 @@
+---
+layout:     post
+title:      Inexor 0.9.0 "Speed Release" announcement
+date:       2017-11-12 20:00 +0100
+author:     a_teammate
+summary:    The Inexor Speed release is finally there.
+---
+
+Hello there everyone, we have big news!
+Inexor 0.9.0@beta is finally released!
+
+Yes it's still unstable software, the switch from "0.8.0-alpha" to "0.9.0@beta" is only a result of our most visible change for end-users:
```

Comment:
Hm, this sentence is not true anymore..
since its 0.9.0@alpha

Maybe
```
The careful reader might have noticed the weird way we changed the version string. Previously it has been "0.8.0-alpha", now it incremented to "0.9.0@alpha".. Wtf happened?
Well, this is the result of our most visible change (yeah really.. "visible"):
```